packages/snaps-controllers/src/types/vendor/global.d.ts

```diff
@@ -1,8 +0,0 @@
-export {};
-
-// eslint-disable-next-line import/unambiguous
-declare global {
```

Comment:
How come the vendor types aren't needed anymore?

Reply:
`DecompressionStream` is now included in TypeScript's on lib files, and the latest version of `@types/readable-stream` has types for `Duplex` and `pipeline`.